src/microsim/schema/simulation.py

```diff
@@ -96,6 +103,15 @@ def _resolve_spaces(self) -> "Self":
     def _xp(self) -> "NumpyAPI":
         return self.settings.backend_module()
 
+    # TODO: binning should happen here like this?
+    @cached_property
```

Comment:
let's just remove this property for now, call `_wavelength_bins()` directly when you need it in `illumination_flux`